src/layout/footer-image/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from "react";
+import { useThrottle } from "@react-hook/throttle";
+import style from "./footer-image.module.css";
+
+type FooterProps<T extends HTMLElement> = {
+  className?: string;
+  container: React.RefObject<T>;
+};
+
+const translateX = (offset: number) => ({
+  transform: `translateX(${offset}px)`,
+});
+const translateY = (offset: number) => ({
+  transform: `translateY(${offset}px)`,
+});
+export default function FooterImage<T extends HTMLElement>({
+  container,
+  className,
+}: FooterProps<T>) {
+  const xOffset = useXMouseOffset(container);
+  const normalizedOffset = Math.max(0.1, Math.log(xOffset)) * 10;
+  const invertedOffset = normalizedOffset * -1;
+
+  return (
+    <div className={className}>
+      <svg
+        width="5195"
+        height="2392"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        className={style.svg}
+      >
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M1682 183.658l60-24.038c60-24.038 180-72.114 300-66.105 120 6.01 240 66.105 360 78.124 120 12.019 240-24.038 360-18.029 120 6.01 240 54.086 300 78.124l60 24.038V370H1682V183.658z"
+          fill="#E61A6B"
```

Comment:
These colors can be substitued with colors from @variant/profile?

Reply:
You are right, my friend! Fixed